controlpanel/frontend/jinja2/tool-list.html

```diff
@@ -9,110 +9,90 @@ <h1 class="govuk-heading-xl">Your tools</h1>
-          </span>
-        </div>
-
-        {# 3rd column with Tool actions #}
-        <div class="tool-actions" style="grid-column: 3/4; align-self: center;" >
-
-          <a class="govuk-button govuk-button--secondary tool-action"
-             data-action-name="open"
-             href="{{ tool_info['url'] }}"
-             rel="noopener"
-             target="_blank"
-             style="margin: 0px"
-             {% if not deployment %} disabled {% endif %}>
-            Open
-          </a>
-
-          <form action="{{ url('restart-tool', kwargs={'name': chart_name}) }}"
-                data-action-name="restart"
-                class="background-submit tool-action"
-                method="post">
-            {{ csrf_input }}
-            <button class="govuk-button govuk-button--secondary right"
-                    data-action-name="restart"
-                    style="margin: 0px"
-                    {% if not deployment %} disabled {% endif %}>
-              Restart
-            </button>
-          </form>
-
-        </div>
-
-      </div >
-
-      {# 2nd row for version selection #}
-      <form action="{{ url('deploy-tool', kwargs={"name": chart_name}) }}"
-            data-action-name="deploy"
-            class="background-submit tool-action"
-            method="post"
-            style="grid-column: 1/2; display: grid; grid-template-columns: 1fr 1fr 1fr;">
-
-        <div style="grid-column: 1/3; align-content:center;">
-          <span>Version:</span>
-
-          <select name="version" class="govuk-select" style="width: 570px">
+{% for chart_name, tool_info in tools_info.items() %}
+{% set deployment = tool_info["deployment"] %}
+<h2 class="govuk-heading-m">{{ tool_info.name }}</h2>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <form action="{{ url('deploy-tool', kwargs={"name": chart_name}) }}"
+        data-action-name="deploy"
+        class="tool-action"
+        id="form-{{ chart_name }}"
+        method="post">
+        {{ csrf_input }}
+      <div class="govuk-form-group">
+          <label class="govuk-label" for="tools">
+            Version
+          </label>
+          <select class="govuk-select govuk-!-width-full govuk-!-font-size-16"
+            data-action-target="{{ chart_name }}"
+            id="tools-{{ chart_name }}" name="version">
             {% set installed_chart_version = None %}
-
-            {# Option for current version (if installed) #}
             {% if deployment %}
               {% set installed_chart_version = deployment.get_installed_chart_version(id_token) %}
               <option class="installed" value="{{ installed_chart_version }}">
-                [{{ installed_chart_version }}]  {{ deployment.get_installed_app_version(id_token) or "Unknown" }} (installed)
+              [Build {{ installed_chart_version }}]  {{ deployment.get_installed_app_version(id_token) or "Unknown" }} (installed)
```

Comment:
Nice one, add "Build" makes a lot of sense!

Reply:
Glad you like it..! :tada: